README.md

```diff
@@ -144,6 +144,9 @@ $ xxd -g 2 -l 64 -p /dev/random | tr -d '\n'
 * `IMGPROXY_CONCURRENCY` — the maximum number of image requests to be processed simultaneously. Default: double number of CPU cores;
 * `IMGPROXY_MAX_CLIENTS` — the maximum number of simultaneous active connections. Default: `IMGPROXY_CONCURRENCY * 5`;
 * `IMGPROXY_TTL` — duration in seconds sent in `Expires` and `Cache-Control: max-age` headers. Default: `3600` (1 hour);
+* `IMGPROXY_USE_ETAG` — boolean. Shows whether a response should be cached or not. If the variable is not presented, then ImgProxy will always return an
```

Comment:
The description seems to be bulky and not clear. I think this would be enough:

> when set, enables using [ETag](https://en.wikipedia.org/wiki/HTTP_ETag) header for the cache control. Default: unset;

Reply:
`when true, enables using [ETag](https://en.wikipedia.org/wiki/HTTP_ETag) header for the cache control. Default: false;`